branding/css/susemanager-login.less

```diff
@@ -67,22 +67,22 @@ body.login-page {
   }
   input[type="submit"] {
     background: @green;
-    color: #fff;
+    color: @white;
     margin-bottom: 1em;
     border: 0;
-    box-shadow: 0px 1px 0px 0px #1a9590, 0px 0px 0px 0px #004744 inset;
-    -moz-box-shadow: 0px 1px 0px 0px #1a9590, 0px 0px 0px 0px #004744 inset;
-    -webkit-box-shadow: 0px 1px 0px 0px #1a9590, 0px 0px 0px 0px #004744 inset;
+    box-shadow: 0px 1px 0px 0px @green, 0px 0px 0px 0px @green-dark inset;
```

Comment:
I understand this was here before, but maybe we can improve this. What's `0px 0px 0px 0px color inset`? It should have no effect unless I'm misunderstanding the intent here?

Reply:
I had a *wtf* moment by looking at what it does live in the page actually. And I just realize that rule is staling because in the login page we don't have any `input[type=submit]`. I will drop completely the rule now.
That was a good catch anyway, I believe there was no reason to have the second part of the `box-shadow` rule. Maybe historically we had to override something else, I can't remember now. :+1: